src/pages/OfferNew/Introduction/Sidebar/mock.ts

```diff
@@ -0,0 +1,24 @@
+import { InsuranceType } from 'utils/insuranceDomainUtils'
+
+export interface InsuranceCompany {
+  name: string
+  switchable: boolean
+}
+
+export const otherInsuranceCompanies: InsuranceCompany[] = [
+  { name: 'Folksam', switchable: true },
```

Comment:
We might be able to use the `PreviousInsurer`-type here if we're really, really lucky. I'll investigate and get back to you

Reply:
Nice! Yes, we'll leave it like this for now and fix this when we know how the previous insurer will be handled!